contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -687,10 +691,10 @@ library ValidationLogic {
   }
 
   /**
-   * @notice Validates if an asset can be activated as collateral in the following actions: supply, transfer,
-   * set as collateral, mint unbacked, and liquidate
+   * @notice Validates if an asset can be activated as collateral in the following actions: set as collateral, mint unbacked
```

Comment:
`mintUnbacked` should also use `validateAutomaticUseAsCollateral`.
It should behave similar to a supply.

Reply:
We opted to not add this here, as minUnbacked is already performed by a trusted/permissioned entity.

Reply:
The bridge is a permissioned entity, but not using `validateAutomaticUseAsCollateral` will probably limit the functioning of the bridge (mintUnbacked on behalf of another user for example).

Apart from that, since `mintUnbacked` is a deferred supply, it's more natural to make it work as a `supply`. The permissioned entity can have the `ISOLATED_ASSET_SUPPLIER` role too.

Reply:
fine, will change